src/applications/check-in/components/pages/Edit/shared/Header.jsx

```diff
@@ -29,6 +29,6 @@ export default function Header(props) {
 
 Header.propTypes = {
   editingPage: PropTypes.string.isRequired,
-  value: PropTypes.string.isRequired,
+  value: PropTypes.oneOfType([PropTypes.string, PropTypes.object]),
```

Comment:
Good point, I think that the value for an empty address is an empty string instead of an empty object, but Shouldn't assume that. I'll revert this back and do a more comprehensive check in the component and pass a string.